helm-chart/binderhub/templates/deployment.yaml

```diff
@@ -30,9 +30,7 @@ spec:
         {{- end }}
         {{- end }}
       annotations:
-        # This lets us autorestart when the configmap changes!
```

Comment:
Can we keep this comment?

Reply:
Looks like this comment is still missing.